app/main/Mindfulness.tsx

```diff
@@ -3,39 +3,32 @@
 "use client";
 
 import '../css/Mindfulness.css'; // Import CSS file
+import { videoList } from '@/lib/data'; // Import video data
 
 interface VideoPlayerProps {
   videoId: string;
-  Label: string;
+  label: string;
 }
 
-const generateVideoPlayer = ({ videoId, Label }: VideoPlayerProps) => (
-  <div className="video-wrapper">
+const generateVideoPlayer = ({ videoId, label }: VideoPlayerProps) => (
+  <div className="video-wrapper" key={label}>
     <iframe
       className='video'
       title='Youtube player'
```

Comment:
This title seems useless. Remove or update?

Reply:
Updated it to video name.